buildscripts/make_dependencies.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Build protoc & netty
+set -ev
+
+# Make protoc
+pushd .
+cd /tmp
+git clone https://github.com/google/protobuf.git
```

Comment:
No make check is fine with me.

sudo-free has to actually be specified. Right now you aren't enabling it:
sudo: false